app/controllers/stories_controller.rb

```diff
@@ -29,6 +29,12 @@ def create
     end
   end
 
+  def destroy
+    @story = current_user.stories.find(params[:id])
+    @story.destroy
+    redirect_to dashboard_path
```

Comment:
What do you think of adding a notice so visual feedback appears after destroying?

```suggestion
    redirect_to dashboard_path, notice: I18n.t('story_deleted_successfully')
```

You would need to add the translation keys too.